eng/push-tarball.sh

```diff
@@ -129,6 +129,16 @@ pushd "$vmr_path"
     git push -u upstream "$new_branch_name"
 
     echo "Creating PR from $new_branch_name to $target_branch"
-    curl -H 'Content-Type: application/json' -d "$data" "$pr_url"
+    curlResult=$(curl -H 'Content-Type: application/json' -d "$data" "$pr_url" 2>/dev/null)
+    webUrl=$(jq -r '.repository.webUrl | values' <<< "$curlResult")
+    pullRequestId=$(jq -r '.pullRequestId | values' <<< "$curlResult")
+    if [ -n "$webUrl" ] && [ -n "$pullRequestId" ]; then
+      echo "$webUrl"/pullrequest/"$pullRequestId"
+    elif [[ "$curlResult" == *"TF401179"* ]]; then
+      "$(jq -r '.message' <<< "$curlResult")"
```

Comment:
What does the message say in this case?

Should we just print a warning instead that the release PR already exists? You can print warning the same way the error is printed

Reply:
This the message:
"TF401179: An active pull request for the source and target branch already exists."
Sure, I will print the same message as you mentioned.